tests/Example/custom-headers/teapot.php

```diff
@@ -2,6 +2,6 @@
 
 $headers = getallheaders();
 
-if ($headers['X-One'] === 'Teapot' && $headers['X-Two'] === 'Pottea') {
+if ($headers['x-one'] === 'Teapot' && $headers['x-two'] === 'Pottea') {
```

Comment:
This causes a test fail as the headers have capitals in the customer headers test -- we should normalize them